tests/classification/test_precision_recall.py

```diff
@@ -195,7 +195,7 @@ def test_no_support(metric_class, metric_fn):
 )
 class TestPrecisionRecall(MetricTester):
 
-    @pytest.mark.parametrize("ddp", [False])
+    @pytest.mark.parametrize("ddp", [False, True])
```

Comment:
@SkafteNicki @Borda do we have multi-gpu tests here to run ddp?

Reply:
yes, we run on the very same cluster as PL

Reply:
TM: https://github.com/PyTorchLightning/metrics/blob/935994c8d56a6c9b2c97db6b86dcd224907b55e4/azure-pipelines.yml#L26

PL: https://github.com/PyTorchLightning/pytorch-lightning/blob/aaccbeea2beb3b903428643d2ee520d2dd05b348/azure-pipelines.yml#L26